src/routes/(proposals)/proposal/[slug]/proposal-page.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+  import clsx from 'clsx';
+  import type { ProposalStoryblok } from '$types/bloks';
+  import { slugify } from '$lib/utils/paths';
+  import { createPackageTimelineData, createRateTimelineData } from '$lib/utils/proposals';
+  import RichText from '$components/rich-text.svelte';
+  import ProposalHero from './proposal-hero.svelte';
+  import ProposalNavigation from './proposal-navigation.svelte';
+  import ProposalDeliverables from './proposal-deliverables.svelte';
+  import ProposalScope from './proposal-scope.svelte';
+  import ProposalTeam from './proposal-team.svelte';
+  import ProposalEstimates from './proposal-estimates.svelte';
+  import ProposalPackage from './proposal-package.svelte';
+  import ProposalTimeline from './proposal-timeline.svelte';
+  import ProposalPackageReply from './proposal-package-reply.svelte';
+  import ProposalRateReply from './proposal-rate-reply.svelte';
+
+  export let proposal: ProposalStoryblok;
+
+  let versions = proposal?.versions || [];
+  let version = versions.length > 0 ? versions[0].version_name : '';
+
+  $: content = versions.find((v) => v.version_name === version);
+  $: type = content?.component === 'proposal-version-package' ? 'package' : 'rate';
+  $: sections = (content?.body || []).map((section) => section.title || '').filter(Boolean);
+
+  $: timelineData =
+    type === 'rate'
+      ? createRateTimelineData(content?.estimates || [], content?.team || [])
+      : createPackageTimelineData(content?.deliverables || [], content?.team || []);
+</script>
+
+{#key version}
+  <ProposalNavigation
+    {sections}
+    versions={versions.map((version) => version.version_name)}
+    bind:version
+  />
+
+  <ProposalHero {proposal} date={content?.date} />
+
+  {#each content?.body || [] as section}
+    <section id={slugify(section.title)} class="my-10 md:my-14 lg:my-20">
+      <div
+        class={clsx(
+          'container mx-auto px-container',
+          'grid grid-cols-1 md:grid-cols-[1fr_2fr_1fr] gap-10 lg:gap-12'
+        )}
+      >
+        <h2 class="text-4xl">{section.title}.</h2>
+        <RichText doc={section.body} />
+      </div>
```

Comment:
Check these sections color is monotone (description should be secondary) and semantics are wrong `h2` to `h4`
![image](https://github.com/significa/significa.co/assets/98809371/f5b2168e-d9ac-42ed-930f-584351ea992e)

Reply:
@nunopolonia this is still unresolved description is still primary, should be secondary:
![image](https://github.com/significa/significa.co/assets/98809371/d48aba19-3537-47e5-97fd-70ab99c6603c)